bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/handler/BlueZBridgeHandler.java

```diff
@@ -88,8 +88,13 @@ public BlueZBridgeHandler(Bridge bridge) {
 
     @Override
     public void initialize() {
+        BluetoothManager manager;
         try {
-            BluetoothManager.getBluetoothManager();
+            manager = BluetoothManager.getBluetoothManager();
+            if (manager == null) {
+                logger.warn("BlueZ Manager returned is null");
+                throw new IllegalStateException();
```

Comment:
In any case, give a reason when throwing an exception.

Reply:
I don't think it's needed here. We catch the exception four lines later and re-throw it with an explanation.

Reply:
Hmmm, yes... Perhaps it makes more sense to rethrow the same exception in the catch block when the message doesn't match. Anyways, we now miss the reason that we received a null value where we expected a manager.

Reply:
I kinda agree with @gedejong what about:

```java
throw new IllegalStateException("BlueZ Manager returned is null");
```

I don't think it's helpful to log a warning and throw an exception, one should be enough

Reply:
Will not work. This exception will be catched by RuntimeException and re-thrown with another message.

Reply:
Yes it will because it passed as the cause of the exception

Reply:
```java
throw new IllegalStateException("Cannot access BlueZ layer.", e);

Reply:
Ok, will add that.